src/coreclr/src/vm/excep.cpp

```diff
@@ -10877,7 +10877,7 @@ void EHWatsonBucketTracker::CaptureUnhandledInfoForWatson(TypeOfReportedError to
     CONTRACTL
     {
         NOTHROW;
-        GC_NOTRIGGER;
```

Comment:
This is called from `SetupWatsonBucketsForEscapingPreallocatedExceptions`. Change the contract on `SetupWatsonBucketsForEscapingPreallocatedExceptions` to `GC_TRIGGERS` too?

Reply:
Also, `GetWatsonBucketTrackerForPreallocatedException`

Reply:
yup, fixed. Will also run outerloop tests -- hoping that has a case for PreallocatedExceptions.